library/spark/spark-accumulo-library/src/main/java/uk/gov/gchq/gaffer/sparkaccumulo/operation/handler/scalardd/GetRDDOfAllElementsHandler.java

```diff
@@ -45,13 +85,139 @@
                                      final Context context,
                                      final AccumuloStore accumuloStore)
             throws OperationException {
+        final String useRFileReaderRDD = operation.getOption(USE_RFILE_READER_RDD);
```

Comment:
Please could you update the GetJavaRDDOfAllElementsExample to tell users about this magic option?

Reply:
Done.